tests/clients/test_experiment.py

```diff
@@ -169,6 +195,41 @@ def test_create_run_schema(started_at, artifact_location):
     }
 
 
+def test_experiment_run_schema():
+    data = ExperimentRunSchema().load(EXPERIMENT_RUN_BODY)
+    assert data == EXPERIMENT_RUN
+
+
+def test_experiment_run_metric_schema():
+    data = MetricSchema().load(METRIC_BODY)
+    assert data == METRIC
+
+
+def test_experiment_run_param_schema():
+    data = ParamSchema().load(PARAM_BODY)
+    assert data == PARAM
+
+
+def test_experiment_run_tag_schema():
+    data = TagSchema().load(TAG_BODY)
+    assert data == TAG
+
+
+def test_experiment_run_data_schema():
```

Comment:
We're actually using the `ExperimentRunDataSchema` to `dump()`, not `load()`. Can we invert these tests to test that?

Reply:
👍